packages/flutter_test/lib/src/window.dart

```diff
@@ -81,6 +82,12 @@ class TestWindow implements ui.SingletonFlutterWindow {
     onMetricsChanged?.call();
   }
 
+  /// Just [devicePixelRatioTestValue], but automatically reset after the test finishes
+  set devicePixelRatioTestValueAutoClear(double value) { // ignore: avoid_setters_without_getters
```

Comment:
Add a doc reference to this new auto clear method in the docs for `devicePixelRatioTestValue`.